bedrock/security/templates/security/bug-bounty/web-eligible-sites.html

```diff
@@ -99,6 +99,23 @@ <h3>Firefox Updates (AUS/Balrog)</h3>
     <li>aus5.mozilla.org</li>
   </ul>
 
+  <h3>Lando</h3>
+  <ul class="mzp-u-list-styled">
+    <li>api.lando.services.mozilla.com</li>
+    <li>api-private.lando.services.mozilla.com</li>
+    <li>lando.services.mozilla.com</li>
+  </ul>
+
+  <h3>Location Services</h3>
+  <ul class="mzp-u-list-styled">
+    <li>location.services.mozilla.com</li>
+  </ul>
+
+  <h3>Normandy</h3>
+  <ul class="mzp-u-list-styled">
+    <li>self-repair.mozilla.org</li>
```

Comment:
I think normandy is 100% served via kinto nowadays, so the URL would be `https://firefox.settings.services.mozilla.com/v1/buckets/main/collections/normandy-recipes/records` but @mythmon can confirm

Reply:
Some very old clients pull from self-repair.mozilla.org. An intermediate age of browser pulls from normandy.cdn.mozilla.net. New versions of Firefox pull from Remote Settings. All three methods are actively used today, since we have a significant population of old users.

Reply:
firefox.settings.services.mozilla.com is already on the list, so that doesnt need to be added.